client/blocks/reader-post-card/style.scss

```diff
@@ -138,8 +138,15 @@
 	&.is-compact {
 		.reader-featured-image {
 			width: 200px !important;
-			min-height: 140px;
-			max-height: 140px;
```

Comment:
I couldn't find a reason for these--the `height` property is already set to `140px`, no matter the original image size.